Sources/CocoaLumberjack/include/DDFileLogger.h

```diff
@@ -534,22 +534,15 @@ extern unsigned long long const kDDDefaultLogFilesDiskQuota;
 // "mylog"     -> "mylog.archived"
```

Comment:
I think we should adjust the comments here. Maybe even remove this API from the public header, since it should no longer be used by clients.

Reply:
BTW, should we keep -renameFile: method in the public API?

Reply:
I don't think they do any harm, so yea. `- reset` will simply reset the lazy vars, which is fine. `-renameFile:` is also not dangerous, as long as the user is aware that this might create a new log file - which might actually be what the user wants. Imagine someone wanting to "mark" a log file by first rolling the log file and then renaming the old one. This would also potentially "save" the log file from being removed at a later point.